Readme.md

```diff
@@ -70,7 +70,8 @@ Welcome all of you future star candidates who want to strive to attend the acade
   > As an academician.  
   > Website link -> [https://portofolio-radif.vercel.app/](https://portofolio-radif.vercel.app/)
 - [Muhammad Favian Revaldy](https://github.com/lukarukikato) [FAV]
-  > As an academician.
+   > As an academician.  
```

Comment:
Next time you should check if the whitespacing is correct or not, but its fine for now since github could still render it properly

Reply:
My apologies again, it must've slipped by. I'll be sure to be more careful next time.